starter-code/basic-algorithms.js

```diff
@@ -1,7 +1,80 @@
 // Names and Input
-
+Abi Summers Github abisummers
+Manon Salaun Github MSLN8
+// Paste your code here
+var hacker1 = "Abi"
+console.log ("the drive's name is "+ hacker1 )
+var hacker2= "Manon"
+console.log ("the navigator's name is " + hacker2 )
 
 //Conditionals
 
+if (hacker1.length>hacker2.length) {
+    console.log ("the driver has the longest name, it has " + hacker1.length)
+}
+    else if (hacker1.length<hacker2.length) {
+        console.log ("yo navigator got the longest name, it has " + hacker2.length + " characters !!")
+    }
+    
+    else if (hacker1.length===hacker2.length) {
+        console.log ("wow, you both got equally long names " + hacker1.length + "!!")
+    }
+    
+    console.log(hacker1.toUpperCase().split("").join(" "))
+
+
+var reverse = ""
+console.log(hacker1.split("").reverse().join(""))
+
+var hacker1 = "Abi" 
+var hacker2 = "Manon"
+
+var one = (hacker1[0])
+var two = (hacker2[0])
+
+
+var final = one.localeCompare(two)
+
+if (final === -1 ) {
+ console.log("The driver's name goes first")
+} 
+else if (final === 1) {
+ console.log("Yo, the navigator goes first definitely")
+}
+else  {
+ console.log("you both have the same name")
+}
+
+var string = prompt("Write a palindrome");
+
+
+var string = string.split(" ").join("").toLowerCase();
```

Comment:
1.You shouldn't declare a variable twice, if you want to apply a method on the string you just prompted, you can just do it like that : 
var myString = prompt("Write a palindrome");
myString = myString.toLowerCase();
console.log(myString);

2. The join method is the opposite of the split method, it cancels the effect of the first method, I am not sure what you were trying to do here ? :)

3. Also, it is not advised to call a variable name by a JavaScript type name, you'll see later that it will cause problems in the code execution, you can call it for example myString :)